app/Http/Controllers/Api/CampaignsController.php

```diff
@@ -54,7 +54,7 @@ public function index(Request $request)
     /**
      * Display the specified resource.
      *
-     * @param  string $id - Rogue ID
+     * @param  string $id - Northstar ID
```

Comment:
```suggestion
     * @param  string $id - Northstar campaign ID
```

Since we (unfortunately) got into the habit of calling user ID's "Northstar IDs", let's be super clear here!